src/components/workspace/controlBar/chapterSelect.tsx

```diff
@@ -38,10 +39,35 @@ export function ChapterSelect(props: ChapterSelectProps) {
   );
   const ChapterSelectComponent = Select.ofType<IChapter>();
 
+  const chapterSelector = (currentChap: number, currentVariant: Variant, isClickable: boolean) => {
+    const stopPopup = (e: any) => {
```

Comment:
This seems like a very primitive approach to intercepting all mouse events, and the typing here of the event as `any` opens the possibility of invoking `stopPropagation` on a non-event.

The Blueprint [`Button`](https://blueprintjs.com/docs/#select/select-component.javascript-api) and [`Select`](https://blueprintjs.com/docs/#select/select-component.javascript-api) components allow you to specify a `disabled` prop for if that component should respond to mouse events. Using both of these props (needed since otherwise the event will bubble) allows us to get rid of the wrapping `<div>`.

The final result looks like this, and attempting to hover over it shows the disabled mouse pointer. From a UX perspective, this is probably better as it doesn't give the impression to students they can click on the chapter select button.

![image](https://user-images.githubusercontent.com/44989315/83448832-25341f80-a485-11ea-85c1-3ad0e9fd0ba6.png)